test/e2e/suite/src/test/resources/cases/ral/ral-integration-queryable.xml

```diff
@@ -14,114 +15,60 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<integration-test-cases>
-    <test-case sql="SHOW READWRITE_SPLITTING HINT STATUS">
-        <assertion expected-data-file="default_readwrite_splitting_hint.xml" />
-        <assertion expected-data-file="set_readwrite_splitting_hint.xml">
-            <initial-sql sql="SET READWRITE_SPLITTING HINT SOURCE = write" />
-            <destroy-sql sql="CLEAR READWRITE_SPLITTING HINT" />
-        </assertion>
-        <assertion expected-data-file="default_readwrite_splitting_hint.xml">
-            <initial-sql sql="SET READWRITE_SPLITTING HINT SOURCE = write;CLEAR READWRITE_SPLITTING HINT" />
-        </assertion>
-        <assertion expected-data-file="default_readwrite_splitting_hint.xml">
-            <initial-sql sql="SET READWRITE_SPLITTING HINT SOURCE = write;CLEAR HINT" />
-        </assertion>
-    </test-case>
-
-    <test-case sql="SHOW SHARDING HINT STATUS">
-        <assertion expected-data-file="default_sharding_hint.xml" />
-        <assertion expected-data-file="set_database_value_hint.xml">
-            <initial-sql sql="SET SHARDING HINT DATABASE_VALUE = 100" />
-            <destroy-sql sql="CLEAR SHARDING HINT" />
-        </assertion>
-        <assertion expected-data-file="add_database_value_hint.xml">
-            <initial-sql sql="ADD SHARDING HINT DATABASE_VALUE t_user_item= 100" />
-            <destroy-sql sql="CLEAR SHARDING HINT" />
-        </assertion>
-        <assertion expected-data-file="add_table_value_hint.xml">
-            <initial-sql sql="ADD SHARDING HINT TABLE_VALUE t_user_item = 100" />
-        </assertion>
-        <assertion expected-data-file="default_sharding_hint.xml">
-            <initial-sql sql="ADD SHARDING HINT TABLE_VALUE t_user_item= 100;CLEAR SHARDING HINT" />
-        </assertion>
-        <assertion expected-data-file="default_sharding_hint.xml">
-            <initial-sql sql="ADD SHARDING HINT TABLE_VALUE t_user_item= 100;CLEAR HINT" />
-        </assertion>
-    </test-case>
 
+<integration-test-cases>
```

Comment:
Can you split this file to `ral-integration-preview.xml` and `ral-integration-show.xml`